inc/class-wpseo-replace-vars.php

```diff
@@ -1281,6 +1351,16 @@ private static function set_basic_help_texts() {
 			new WPSEO_Replacement_Variable( 'searchphrase', __( 'Search phrase', 'wordpress-seo' ), __( 'Replaced with the current search phrase', 'wordpress-seo' ) ),
 			new WPSEO_Replacement_Variable( 'term_hierarchy', __( 'Term hierarchy', 'wordpress-seo' ), __( 'Replaced with the term ancestors hierarchy', 'wordpress-seo' ) ),
 			new WPSEO_Replacement_Variable( 'sep', __( 'Separator', 'wordpress-seo' ), $separator_description ),
+			new WPSEO_Replacement_Variable( 'currentdate', __( 'Current date', 'wordpress-seo' ), __( 'Replaced with the current date', 'wordpress-seo' ) ),
```

Comment:
I wonder if we should make this translatable when it's hidden to users.

Reply:
I think that it would increase the complexity of the PR with no real benefit :)

Reply:
But doesn't every function that is executed have some impact on performance?
Even though the impact is probably tiny here, performance is one of our features too.

Reply:
I think it's @Xyfi 's call here. I think we're pressed on time to implement this as well, but I have no serious reservation if we agree to deal with that too :)